client/src/components/Ratings/Ratings.jsx

```diff
@@ -54,6 +54,8 @@ const RatingSection = styled.section`
   background-color: ${({ theme }) => theme.colors.light};
   display: flex;
   flex-direction: row;
-  padding: 2rem 0rem 4rem 1rem;
   color: ${({ theme }) => theme.colors.secondary};
+  @media (min-width: 768px) { height: 630px;  }
+  @media (max-width: 768px) { width: 100%; flex-wrap: wrap;}
```

Comment:
Responsive is looking good! Would require some changes to how you're doing your ratings bars, but future improvement might be to make the bars span full width if it's in a smaller breakpoint.

![Screen Shot 2022-04-14 at 1 04 39 PM](https://user-images.githubusercontent.com/67052402/163467342-e2ec8047-b3ab-4ee3-8920-fe08ff72c3ea.png)

Reply:
Will take a look